service/bookturks/user_views/user_resetPassword_view.py

```diff
@@ -0,0 +1,112 @@
+from django.contrib.auth.tokens import default_token_generator
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode, urlsafe_base64_decode
+from django.template import loader
+from django.conf import settings
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+from django.http import Http404, HttpResponseRedirect
+from django.shortcuts import render
+from django.core.urlresolvers import reverse
+from sendgrid.helpers.mail import *
+
+from service.bookturks.adapters.UserAdapter import UserAdapter
+from service.bookturks.alerts import init_alerts, set_alert_session
+from service.bookturks.Constants import DANGER, SUCCESS, RESET_PASSWORD_CONFIRM, SERVICE_MAIN_HOME
+
+import sendgrid
+
+
+def get_username_password_reset_view(request):
+    request, alert_type, alert_message = init_alerts(request=request)
```

Comment:
We do not need this line since this view does not render any page but just redirects.